packages/reducers/src/makeEnhancer.js

```diff
@@ -8,6 +23,29 @@ import composeReducers from './composeReducers';
 
 export const storeInterface = makeStoreInterface('reducers');
 
+const cleanUpReducer = (state, action) => {
+	if (action.type !== '@redux-tools/REDUCERS_EJECTED') return state;
+
+	const namespaceGroup = action.meta.feature || DEFAULT_FEATURE;
+
+	const pathToNamespace = [namespaceGroup, action.meta.namespace];
+	const pathToReducer = when(always(action.meta.namespace), concat(pathToNamespace))([]);
```

Comment:
unnecessary complexity, ˙action.meta.namespace ? pathToNamespace : []˙
 
this will also fail if action.meta is not defined, we have util functions for `getNamespaceByAction` and maybe even `getFeatureByAction`, use them!